migrations/2022_05_08_031822_create_stats_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('stats', function (Blueprint $table) {
+            $table->id();
+            $table->integer('points');
+            $table->integer('player_id')->unsigned();
```

Comment:
you need to set this foreign key to the correct table

Reply:
For this to work I guess you will need the players table to be created first I guess

Reply:
done